CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 # Changelog
 
 ## [Unreleased]
+* [Developer/UI]: Refreshed the create new user account page.
```

Comment:
Can you add a link to the this PR for this changelog entry. Like I did below for the 22.05.1 hotfix release